src/shortcuts.ts

```diff
@@ -282,7 +282,7 @@ export const inputHandlers = (store: Store<State, any>) => ({
     shortcutEmitter.trigger('shortcut', shortcut)
 
     if (!shortcut.canExecute || shortcut.canExecute(store.getState())) {
-      e.preventDefault()
+      if (shortcut.id !== 'copyCursor') e.preventDefault()
```

Comment:
This is okay as a temporary measure, but we'll have to think about how to expand the Shortcut structure to accommodate this in a more general way. Having shortcut.ts have to know about specific shortcut behavior is not a good idea. We want shortcut-specific behavior to be encapsulated within the appropriate shortcut file.